### PR TITLE
Add Demo for Openstack Cinder CSI Plugin Deployment

### PR DESCRIPTION
**Pull request by rigrassm**
Adding a demo for the Cinder CSI Plugin following the guidelines specified in the newly added [demo/csi/README.md](demo/csi/cinder-csi-plugin/README.md).

### CODE REVIEW

File: demo/csi/cinder-csi-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  - This option must match the `mount_dir` specified in the `csi_plugin` stanza for the task.
+
+- `--cloud-config=/etc/config/cloud.conf`
+  
+  - The location that the cloud.conf file was mounted inside the container
+  
+- `--nodeid=${node.unique.name}`
+
+  - A unique ID for the node the task is running on. Reccomend using `${node.unique.name}`
```
Comment (tgross) [inline demo/csi/cinder-csi-plugin/README.md:37]:
```suggestion
  - A unique ID for the node the task is running on. Recommend using `${node.unique.name}`
```

File: demo/csi/cinder-csi-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- `--cluster=${NOMAD_DC}`
+
+  - The cluster the Controller/Node is apart of. Reccomend using `${NOMAD_DC}`
```
Comment (tgross) [inline demo/csi/cinder-csi-plugin/README.md:41]:
```suggestion
  - The cluster the Controller/Node is a part of. Recommend using `${NOMAD_DC}`
```

File: demo/csi/cinder-csi-plugin/README.md (hunk 1 above)
Comment (rigrassm) [reply demo/csi/cinder-csi-plugin/README.md]: Updated.

File: demo/csi/cinder-csi-plugin/README.md (hunk 2 above)
Comment (rigrassm) [reply demo/csi/cinder-csi-plugin/README.md]: Updated

**Comment by tgross** [comment]
@rigrassm if you squash your changes so that only the commit for the `@rigrassm` account is left, the CLA bot will be satisfied.

**Comment by rigrassm** [comment]
> @rigrassm if you squash your changes so that only the commit for the `@rigrassm` account is left, the CLA bot will be satisfied.

Didn't notice I made that change with my work account! I'm about to get on a plane so I'll try getting that change in on my tablet.

**Comment by rigrassm** [comment]
Was able to get that fixed up on my phone so it should be good to go!

**Comment by tgross** [comment]
cc'ing @angrycub as a heads up

**Pull request merged**